source/_usecases/find the Index of an element in an Array

```diff
@@ -0,0 +1,17 @@
+extends: _layouts.usecase
+date: 2018-03-27
+https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex
```

Comment:
Please use the link for Array.indexOf => https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf

Reply:
also, it needs to be prefixed with `link: `